wally-registry-backend/src/auth.rs

```diff
@@ -80,26 +98,68 @@ async fn verify_github_token(request: &Request<'_>) -> Outcome<WriteAccess, Erro
     };
 
     let client = Client::new();
-    let response = client
-        .get("https://api.github.com/user")
+
+    let org_response = client
+        .get("https://api.github.com/user/memberships/orgs")
```

Comment:
At the moment we don't have permission to do this! We need to add the `read:org` scope to our oauth request in login.rs. It is a space-delimited list so it should look like `"scope": "read:user read:org",`.

As consequence of this is if we ship this change then every user will suddenly have to reauth and grant these new permissions. We need to be careful with this. We automatically deploy all changes to main. That means the registry backend will update and require the new scope however there wouldn't be a new release so users would be unable to auth. I'm also not sure forcing everyone to upgrade their version of wally is the best idea.

I think we should find a way to allow existing logins to work and only check org ownership if they aren't allowed to publish to a scope via the `owners.json`. Ideally we could prompt a user that their oauth token doesn't have enough permission to verify org membership and they should login again.